coverage/htmlfiles/coverage_html.js

```diff
@@ -81,7 +81,31 @@ function sortColumn(th) {
         .forEach(tr => tr.parentElement.appendChild(tr));
 
     // Save the sort order for next time.
-    localStorage.setItem(coverage.INDEX_SORT_STORAGE, JSON.stringify({column, direction}));
+    if (th.id !== "function_or_class") {
+        var th_id = "";
+        const stored_list = localStorage.getItem(coverage.INDEX_SORT_STORAGE);
+        if (stored_list) {
+            ({th_id, direction} = JSON.parse(stored_list))
```

Comment:
I'm a JS newb: are the parens needed around this assignment?

Reply:
Newb here too, so I searched and TIL they are required: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment